rubocop.yml

```diff
@@ -0,0 +1,24 @@
+AllCops:
```

Comment:
I'm confused. I thought we had to have the rubocop.yml file and that's where I got rid of the Style/ClassAndModuleChildren that was giving Tyna trouble. I just included it in this commit because I had added a final empty line to the file.